modules/markup/markdown/goldmark.go

```diff
@@ -184,6 +186,18 @@ func (g *ASTTransformer) Transform(node *ast.Document, reader text.Reader, pc pa
 			if css.ColorHandler(strings.ToLower(string(colorContent))) {
 				v.AppendChild(v, NewColorPreview(colorContent))
 			}
+		case *ast.Emphasis:
+			// check if inside blockquote for attention, expected hierarchy is
+			// Emphasis < Paragraph < Blockquote
+			blockquote, isInBlockquote := n.Parent().Parent().(*ast.Blockquote)
+			if isInBlockquote && !attentionMarkedBlockquotes.Contains(blockquote) {
+				fullText := string(n.Text(reader.Source()))
+				if fullText == AttentionNote || fullText == AttentionWarning {
+					v.SetAttributeString("class", []byte("attention-"+strings.ToLower(fullText)))
+					v.Parent().InsertBefore(v.Parent(), v, NewAttention(fullText))
+					attentionMarkedBlockquotes.Add(blockquote)
```

Comment:
```suggestion
			if isInBlockquote && attentionMarkedBlockquotes.Add(blockquote) {
				fullText := string(n.Text(reader.Source()))
				if fullText == AttentionNote || fullText == AttentionWarning {
					v.SetAttributeString("class", []byte("attention-"+strings.ToLower(fullText)))
					v.Parent().InsertBefore(v.Parent(), v, NewAttention(fullText))
```
`Add` returns `true` if the element was added, `false` otherwise.

Reply:
It'll be too early to add it if `fullText == AttentionNote || fullText == AttentionWarning` is `false`

Reply:
Ah right, missed that.